t/README.md

```diff
@@ -1,72 +1,27 @@
-# Unit Tests for PG
+# Welcome to WeBWorK
```

Comment:
I would keep this as relevant to PG instead of WeBWorK in that these should be testing the PG side of things.  We are trying to further split the WeBWorK side (UI) from the PG.

Reply:
This README.md file seems to be identical to the one at the top level of PG.  Perhaps we should revert to some language about running the tests.

Reply:
My mistake in the splitting of the PR.  I'll find it.